commands/download.js

```diff
@@ -1,14 +1,10 @@
 var chalk = require('chalk')
 var prettyBytes = require('pretty-bytes')
-var Dat = require('dat-js')
 var logger = require('status-logger')
 var speedometer = require('speedometer')
 var ui = require('../lib/ui')
 
-module.exports = function (args) {
-  if (args && args.exit) args.upload = false
```

Comment:
We lost this setting. When `--exit` was specified, the download wouldn't upload data. 

I am fine getting rid of if if you think so too.